mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -1103,28 +1103,39 @@ ParseResult GenericOp::parse(OpAsmParser &parser, OperationState &result) {
 static void getGenericEffectsImpl(
     SmallVectorImpl<SideEffects::EffectInstance<MemoryEffects::Effect>>
         &effects,
-    ValueRange results, const ValueRange inputOperands,
+    LinalgOp linalgOp, ValueRange results, const ValueRange inputOperands,
```

Comment:
Can you remove `inputeOperands` and `outputOperands` from the signature? They can be queried from `linalgOp`.

Reply:
Thanks for your advice. I have remove duplicated parameters from signature.